generator_process.py

```diff
@@ -161,6 +137,41 @@ def writeException(fatal, e):
         writeStr(e)
         stdout.flush()
 
+    try:
+        from absolute_path import absolute_path
+        # Support Apple Silicon GPUs as much as possible.
+        os.environ["PYTORCH_ENABLE_MPS_FALLBACK"] = "1"
+        sys.path.append(absolute_path("stable_diffusion/"))
+        sys.path.append(absolute_path("stable_diffusion/src/clip"))
+        sys.path.append(absolute_path("stable_diffusion/src/k-diffusion"))
+        sys.path.append(absolute_path("stable_diffusion/src/taming-transformers"))
+
+        site.addsitedir(absolute_path(".python_dependencies"))
+        import pkg_resources
+        pkg_resources._initialize_master_working_set()
+
+        from stable_diffusion.ldm.generate import Generate
+        from omegaconf import OmegaConf
+        from PIL import ImageOps
+        from io import StringIO
+    except ModuleNotFoundError as e:
+        min_files = 10 # bump this up if more files get added to .python_dependencies in source
+                       # don't set too high so it can still pass info on individual missing modules
+        if not os.path.exists(".python_dependencies") or len(os.listdir()) < min_files:
+            e = "Python dependencies are missing. Click update to download full addon."
```

Comment:
This needs an else branch so `writeException` works properly
```suggestion
            e = "Python dependencies are missing. Click update to download full addon."
        else:
            e = repr(e)
```